Description: Redirect error message "failed to create directory /home/<user>/.asy" to /dev/null
Origin: Hilmar Preusse <hille42@web.de>
Forwarded: Not yet.
Author: Hilmar Preusse <hille42@web.de>
Last-Update: 20210617

Index: asymptote/tests/Makefile
===================================================================
--- asymptote.orig/tests/Makefile	2021-02-23 23:12:56.491129303 +0100
+++ asymptote/tests/Makefile	2021-06-17 22:55:38.126338667 +0200
@@ -10,11 +10,11 @@
 
 $(TESTDIRS)::
 	@echo
-	../asy -dir ../base $@/*.asy
+	../asy -dir ../base $@/*.asy 2> /dev/null
 
 $(EXTRADIRS)::
 	@echo
-	../asy -dir ../base $@/*.asy
+	../asy -dir ../base $@/*.asy 2> /dev/null
 
 clean:  FORCE
 	rm -f *.eps
